src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts

```diff
@@ -96,10 +97,11 @@ export const useGetTableData = (props: Props) => {
       })
       if (odp) {
         dispatch(DataActions.getOriginalDataPointData({ assessmentName, countryIso, cycleName }))
-        dispatch(
-          DataActions.getNodeValuesEstimations({ assessmentName, countryIso, cycleName, tableName, sectionName })
-        )
+        if (user)
```

Comment:
this should happen if user can edit section data

Reply:
Great, updated!

Reply:
Authorizer.canEditData

Reply:
Push didn't go through